clients/admin-ui/cypress/e2e/home.cy.ts

```diff
@@ -0,0 +1,41 @@
+import { stubHomePage } from "cypress/support/stubs";
+
+describe("Home", () => {
+  beforeEach(() => {
+    cy.login();
+    cy.visit("/");
+    cy.location("pathname").should("eq", "/");
```

Comment:
we don't need this here, do we? since the `auth.cy.ts` test already covers this. maybe just `.login()` and `.visit("/")`?